src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,34 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          {user && <UserInfo user={user} />}
+        </p>
+      </div>
+
+      <p className="PostInfo__body">{body}</p>
+
+      <hr />
```

Comment:
This `hr` is rendered even when there are comments, which is not needed!

Reply:
I dunno if i get it, but i just delete the line 22.